data/battle_scripts_1.s

```diff
@@ -3881,26 +3881,39 @@ BattleScript_EffectDoNothing::
 	attackcanceler
 	attackstring
 	ppreduce
-	jumpifmove MOVE_HOLD_HANDS, BattleScript_EffectHoldHands
 	attackanimation
 	waitanimation
-	jumpifmove MOVE_CELEBRATE, BattleScript_EffectCelebrate
-	jumpifmove MOVE_HAPPY_HOUR, BattleScript_EffectHappyHour
 	incrementgamestat GAME_STAT_USED_SPLASH
 	printstring STRINGID_BUTNOTHINGHAPPENED
 	waitmessage B_WAIT_TIME_LONG
 	goto BattleScript_MoveEnd
-BattleScript_EffectHoldHands:
+
+BattleScript_EffectHoldHands::
+	attackcanceler
+	attackstring
+	ppreduce
 	jumpifsideaffecting BS_TARGET, SIDE_STATUS_CRAFTY_SHIELD, BattleScript_ButItFailed
```

Comment:
why is there a crafty shield check? Could you write a test and remove this check and see if it passes? Because usually those are checked in C

Reply:
Added the test, but removing this line makes it so it doesn't display the "x was protected" message, but still doesn't play the animation 🤔

Reply:
Very weird. `SIDE_STATUS_CRAFTY_SHIELD` is only used here but doesn't matter. The pr is already an improvement and I don't want to go down the rabbit hole figuring it out why this happens for this move